### PR TITLE
Doc block generation clean up. 

### PR DESCRIPTION
**Pull request by Bonscho**
Did not hit all issues, but most of them. 
Reduced the number of errors and warnings resulting from invalid or incomplete doc blocks in a mid size project by 50%.

### CODE REVIEW

**Comment by staabm** [comment]
please revert all changes of filepermission introduced with this PR

File: generator/lib/behavior/nestedset/NestedSetBehaviorPeerBuilderModifier.php [hunk 1]
```diff
@@ -85,6 +85,13 @@ public function staticAttributes($builder)
  */
 const SCOPE_COL = '" . $tableName . '.' . $this->getColumnConstant('scope_column') . "';
 ";
+        }else{
```
Comment (staabm) [inline generator/lib/behavior/nestedset/NestedSetBehaviorPeerBuilderModifier.php]: this change seems to be unrelated

File: generator/lib/behavior/nestedset/NestedSetBehaviorObjectBuilderModifier.php [hunk 2]
```diff
@@ -1439,14 +1442,14 @@ protected function addDeleteDescendants(&$script)
...
- *
  * @return     int 		number of deleted nodes
+ * @throws Exception
  */
 public function deleteDescendants(PropelPDO \$con = null)
 {
     if (\$this->isLeaf()) {
         // save one query
-        return;
+        return 0;
```
Comment (staabm) [inline generator/lib/behavior/nestedset/NestedSetBehaviorObjectBuilderModifier.php]: this might break things

File: generator/lib/behavior/nestedset/NestedSetBehaviorPeerBuilderModifier.php (hunk 1 above)
Comment (Bonscho) [inline generator/lib/behavior/nestedset/NestedSetBehaviorPeerBuilderModifier.php]: Class constants should be available always. It is hard to use them if it is unclear whether they exist or not.

File: generator/lib/behavior/nestedset/NestedSetBehaviorObjectBuilderModifier.php (hunk 2 above)
Comment (Bonscho) [inline generator/lib/behavior/nestedset/NestedSetBehaviorObjectBuilderModifier.php]: Methods having inconsistent return value types shoul not break things.
